_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,433 @@
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility Tree
+  - CSS Styling
+  - DOM Tree
+  - Language
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Daniël Strik
+    - Jean-Yves Moyen
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [text node][] that is either [visible][] or [included in the accessibility tree][].
+
+## Expectation
+
+For each test target that includes at least one of the [visual reference words][], one of the following is true:
+
+- (**non-visual reference**) the test target is on the same [web page][] as a [textual][text] instruction that also identifies that [web content][] by a non-visual characteristic; or
+- (**visible words**) each [visual reference word][] in the test target is included in the [visible text content][] of the identified content; or
+- (**accessible words**) each [visual reference word][] in the test target is included in the [accessible name][] of the identified content; or
+- (**no indication**) the test target does not identify any [web content][] through the use of any [visual reference words][]; or
+- (**no instruction**) the test target does not give instructions about any [web content][] through the use of any of the [visual reference words][].
+
+## Assumptions
+
+- This rule assumes that [visual reference words][] are forms of information conveyed through presentation, because of this, failing this rule fails both [Success Criterion 1.3.1 Info and Relationships][sc131] and [Success Criterion 1.3.3 Sensory Characteristics][sc133]. Presentation is not limited to CSS and includes images such as the image of a circle with text.
+
+- This rule assumes that non-visual users will interpret some visual reference words as meaning "ahead" or "backwards" in the reading order. For example in most contexts "see the content below" will mean ahead in the DOM tree reading order which is not a visual reference and should pass this test. Note however that the DOM tree reading order can be different from the visual order of things, which could result in "see the content below" only referring to the visual order of things in which case it is not a correct reference.
+
+- This rule assumes that the language of each test target can be correctly determined (either programmatically or by analyzing the content), and sufficiently understood.
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+The rule doesn't require the non-visual characteristic description to be included in the accessibility tree. If the alternatives are not included in the accessibility tree, only [Success Criteria 1.3.1 Info and Relationships][sc131] would fail instead of both [Success Criterion 1.3.3 Sensory Characteristics][sc133] and [1.3.1][sc131]. Hence, the rule passes in these cases as it is not a failure of all accessibility requirements.
```

Comment:
```suggestion
The rule doesn't require the non-visual characteristic description to be included in the accessibility tree. If the alternatives are not included in the accessibility tree, only [Success Criteria 1.3.1 Info and Relationships][sc131] would fail instead of both [Success Criterion 1.3.3 Sensory Characteristics][sc133] and [Success Criteria 1.3.1 Info and Relationships][sc131]. Hence, the rule passes in these cases as it is not a failure of all accessibility requirements.
```